packages/common/src/utils/SafeLocalStorage.ts

```diff
@@ -8,7 +8,7 @@ export type SafeLocalStorageItems = {
   '@appkit/connected_social': string
   '@appkit/connected_social_username': string
   '@appkit/recent_wallets': string
-  '@appkit/deeplink_choice': string
+  WALLETCONNECT_DEEPLINK_CHOICE: { href: string; name: string }
```

Comment:
@magiziz I'm confused why we don't follow the same naming pattern?

Reply:
Since universal provider uses `WALLETCONNECT_DEEPLINK_CHOICE` local storage key to initiate a deep link. We could push a PR to universal provider repo to use same naming pattern. Lmk what you think.

Reply:
Okay if that's the case we should add a JSDoc comment on top of this key IMO. That'a a different case.

But can we still keep returning string and do the parsing outside of it?